src/main/java/core/basesyntax/FileWork.java

```diff
@@ -15,10 +15,10 @@
         File file = new File(fileName);
         try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
             StringBuilder builder = new StringBuilder();
-            int value = bufferedReader.read();
-            while (value != -1) {
-                builder.append((char) value);
-                value = bufferedReader.read();
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                builder.append(value).append(DIVIDER);
+                value = bufferedReader.readLine();
```

Comment:
you should close connection after this, you don't use `bufferedReader` in code afterwards. And, you're using [try-with-resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html), you don't need `bufferedReader.close()` to close connection.

Reply:
done!